### ISSUE TITLE
Exercise 2.2 and Figure 2.3 do not match

### ISSUE THREAD

**Issue by laurasailar** [awsccp/awsccp.github.io#4]
Exercise 2.2 says to enter quantities of 2 for the AWS services.  Figure 2.3 shows pricing for single services.

**Comment by dbclinton** [comment]
Good catch. Actually figure 2.3 was meant to be nothing more than a generic example of what output could look like, but I wasn't clear enough about that in the text. I'll try to get that fixed.
Thanks
